### PR TITLE
csv copy and tests

### PR DESCRIPTION
**Pull request by mohitkhullar**

### CODE REVIEW

**Comment by nirbhayc** [comment]
Can we call it 'load' instead of 'csvcopy'?

File: tests/csvcopy.test/t00.expected [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[exec procedure sys.cmd.load('/home/mk/data/mohitdb1/agg.csv', 't1')] failed with rc -3 [sys.load(db,csv,tbl,seperator,h...]:2: File doesn't exist
```
Comment (nirbhayc) [inline tests/csvcopy.test/t00.expected]: Relative path?

Comment (nirbhayc) [reply tests/csvcopy.test/t00.expected]: Also, how does it play with the system's security? Theoretically, one should be able to dump data in password file into a table.. :) I'd restrict the directory from where one can load data.

Comment (mohitkhullar) [reply tests/csvcopy.test/t00.expected]: this is a failure test case, it won't work.. only data files inside db's basedir will work for now

Comment (nirbhayc) [reply tests/csvcopy.test/t00.expected]: Ok, I overlooked that.
```
+    } else if ((strncmp(thedb->basedir, fname, basedir_len) != 0) || (fname[basedir_len] != '/')) {
+        return luaL_error(lua, "File not in database directory");
```
Can we have a test covering that?

Comment (mohitkhullar) [reply tests/csvcopy.test/t00.expected]: Added that test too  t02.req

**Pull request merged**